src/planner/planner.c

```diff
@@ -1375,6 +1375,19 @@ timescaledb_set_rel_pathlist(PlannerInfo *root, RelOptInfo *rel, Index rti, Rang
 #endif
 			TS_FALLTHROUGH;
 		default:
+			/*
+			 * Set the indexlist for a hypertable parent to NIL since we
+			 * should not try to do any index scans on hypertable parents,
+			 * similar to how it works for partitioned tables.
```

Comment:
Is this something that Postgres does for partitioned tables?? If yes then would be good to point to the postgres source code here.

Reply:
It is done inside `get_actual_variable_range`, which is mentioned last in the paragraph.

Reply:
... well, later in the paragraph, not last.